java/src/jmri/jmrix/sprog/sprogslotmon/SprogSlotMonAction.java

```diff
@@ -23,16 +26,21 @@ public SprogSlotMonAction(SprogSystemConnectionMemo memo) {
         this(Bundle.getMessage("SprogSlotMonitorTitle"), memo);
     }
 
+    public SprogSlotMonAction() {
+        super(Bundle.getMessage("SprogSlotMonitorTitle"));
+        _memo = InstanceManager.getNullableDefault(SprogSystemConnectionMemo.class);
+    }
+
     @Override
     public void actionPerformed(ActionEvent e) {
 
         if(_memo.getCommandStation() == null) {
-           // create SlotManager if it doesn't exist
-           _memo.configureCommandStation(_memo.getNumSlots());
+                // create SlotManager if it doesn't exist
+                _memo.configureCommandStation(_memo.getNumSlots());
+            }
+            
+            SprogSlotMonFrame f = new SprogSlotMonFrame(_memo);
+            f.setVisible(true);
         }
 
-        SprogSlotMonFrame f = new SprogSlotMonFrame(_memo);
-        f.setVisible(true);
-    }
```

Comment:
The indentation seems to be messed up for this method. The `if` statement that starts on line 37 seems to end at line 44, but instead ends on line 40.